### PR TITLE
Episode 4: sphinx + readthedocs demo

### PR DESCRIPTION
**Pull request by svenvanderburg**
Adresses #4

### CODE REVIEW

File: _episodes/04-sphinx.md [hunk 1]
```diff
@@ -1,374 +1,106 @@
...
 
-## Group exercise: Build Sphinx documentation using RST
+## Motivation
+What if a README file and in-code documentation is not enough? 
+What if you want a more extensive documentation because you expect more users or collaborators?
 
-We will take the first steps in creating documentation using Sphinx, and learn some
-RST syntax along the way.
+A very popular and nice solution is to use `Sphinx` together with `ReadtheDocs`:
+- `Sphinx` let's you create nicely-formatted HTML pages out of simple markdown files. It is language-independent.
```
Comment (JensWehner) [inline _episodes/04-sphinx.md:19]: What do you mean by language independent. Independent of the markdown used or the programming language?

Comment (svenvanderburg) [reply _episodes/04-sphinx.md]: It is both actually, but I ment programming language. I added that.

**Comment by svenvanderburg** [comment]
@JensWehner apart from your comment do you approve the changes?

**Comment by svenvanderburg** [comment]
OK, for the sake of pushing forward I am just merging this 🤟

**Pull request merged**